### PR TITLE
Fix: Added corect Marker Location and animation in the Help Section

### LINKED ISSUE #546: Add correct Marker Location in Help
**Summary:**

The map in the Help Section, shows 4 markers which are not properly labeled, and by default zoom in inside the map.

**Steps to reproduce:**

Open Navigation Drawer > Help > Find Locations

**Expected behavior:**

Marker should point to correct address (written above) and should be zoomed in near that address

**Observed behavior:**

Instead, there are 4 markers (not pointing to correct address) and map is not zoomed in

**Device and Android version:**

Xiaomi Marshmallow 

**Screenshots:**

(Markers are pointing to the wrong address and are 4 of them)
(No Zoom In)

### PR DESCRIPTION
**Pull request by miPlodder**
Fixes #546 

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by miPlodder** [comment]
@therajanmaurya @dilpreet96 Please review this PR  ?

**Comment by dilpreet2028** [comment]
Good work @miPlodder , as there is just 1 `LatLng` so remove the list and use an object.

**Comment by miPlodder** [comment]
@dilpreet96 I have updated the PR.

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/LocationsFragment.java [hunk 1]
```diff
@@ -67,22 +67,28 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+                .title(getString(R.string.mifos_initiative)));
+
+    }
+
+    private void addAnimationToHeadquarter(GoogleMap googleMap, LatLng headquarterLatLng) {
+
+        googleMap.animateCamera(CameraUpdateFactory.newLatLngZoom(headquarterLatLng, 16.0f));
     }
 
     private List<LatLng> getLatLngList() {
```
Comment (dilpreet2028) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/LocationsFragment.java]: @miPlodder , as this function is not used anywhere so can you remove this.

**Review by dilpreet2028 — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/LocationsFragment.java (hunk 1 above)
Comment (miPlodder) [reply app/src/main/java/org/mifos/mobilebanking/ui/fragments/LocationsFragment.java]: @dilpreet96  Will remove it and update the PR.

**Comment by dilpreet2028** [comment]
Good work! @miPlodder

**Review by dilpreet2028 — APPROVED** [review]

**Pull request merged**